.gitmodules

```diff
@@ -7,8 +7,16 @@
 [submodule "l1-contracts/lib/openzeppelin-contracts-upgradeable"]
```

Comment:
I think we should name it `l1-contracts/lib/openzeppelin-contracts-upgradeable-v5`. At some point they will upgrade to v6 and what we will do? Again making incompatibles hacks?

Reply:
we had a discussion about it with txfusion who had suggested this solution but we can push back on that